integreat_cms/core/management/commands/repair_tree.py

```diff
@@ -0,0 +1,88 @@
+import logging
+
+from django.core.management.base import CommandError
+
+from ....cms.models import Page
+from ..log_command import LogCommand
+
+logger = logging.getLogger(__name__)
+
+
+class Command(LogCommand):
+    """
+    Management command to repair a broken Treebeard page tree
+    """
+
+    help = "Search & replace links in the content"
+
+    def add_arguments(self, parser):
+        """
+        Define the arguments of this command
+
+        :param parser: The argument parser
+        :type parser: ~django.core.management.base.CommandParser
+        """
+        parser.add_argument(
+            "--page", help="The ID of a page belongig to the broken tree."
+        )
+        parser.add_argument(
+            "--commit",
+            action="store_true",
+            help="Whether changes should be written to the database",
+        )
+
+    # pylint: disable=arguments-differ
+    def handle(self, *args, page, commit, **options):
+        r"""
+        Try to run the command
+
+        :param \*args: The supplied arguments
+        :type \*args: list
+
+        :param page: The page ID (node) of a broken tree
+        :type page: str
+
+        :param commit: Whether changes should be written to the database
+        :type commit: bool
+
+        :param \**options: The supplied keyword options
+        :type \**options: dict
+
+        :raises ~django.core.management.base.CommandError: When the input is invalid
+        """
+        if not page:
+            raise CommandError
+        root_node = Page.objects.get(id=page).get_root()
+        self.calculcate_left_right_values(root_node, 1, commit)
```

Comment:
```suggestion
        try:
            tree_id = Page.objects.get(id=page_id).tree_id
        except Page.DoesNotExist as e:
            raise CommandError(f'The page with id "{page_id}" does not exist.') from e
        try:
            root_node = Page.objects.get(tree_id=tree_id, parent=None)
        except (Page.DoesNotExist, Page.MultipleObjectsReturned) as e:
            raise CommandError(
                f'The root page of tree id "{tree_id}" cannot be determined: {e}'
            ) from e
        action = "Fixing" if commit else "Detecting problems in"
        self.print_info(f"{action} tree with id {tree_id}...")
        self.pages_seen = []
        self.calculate_left_right_values(root_node, 1, commit)
        self.check_for_orphans(tree_id)
```
We could improve the error messages here in case the input does not make sense.
Also, the `get_root()` uses the tree fields itself, so I would suggest to rather check for `parent=None` instead.